Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -647,7 +649,7 @@ def get_domain_reputation(client: Client, score_calc: DBotScoreCalculator, domai
                                                                FeedIndicatorType.Domain
                                                                )
         if intelligence_relationships:
-            relationships.append(intelligence_relationships)
+            relationships.extend(intelligence_relationships)
```

Comment:
please remove the # pragama: no cover in this block (line 646) and update your test to run the test where threat_model_association param is True.

Reply:
btw, if you were using typing (e.g. specify the return type of get_intelligence method) mypy would have found this issue for you when you developed it.